pkg/cache/cache.go

```diff
@@ -82,23 +92,12 @@ type Response struct {
 	Canary bool
```

Comment:
I think we can get rid of this until we have that feature implemented

Reply:
This was explicitly requested by @junr03, so making provision to implement it.

Reply:
They are not using the discovery canary field. They are using EDS canary field, which is already covered by the fact that we are using the eds proto. cc @junr03

Reply:
Sorry @kyessenov I don't remember talking to you about this? We don't currently use this field in the response.

Reply:
Awesome, then I'll drop it for now. I remember on slack that RDS + canary is something that we want to support.